dbt/models/sale/docs.md

```diff
@@ -1,19 +1,67 @@
+# flag
+
 {% docs flag %}
-This table holds the flag information from the sales val program.
+PIN-level sales validation flags created by
+[model-sales-val](https://github.com/ccao-data/model-sales-val).
+
+This is the primary sales validation output table. Flags within this table
+should be possible to reconstruct using the other sales validation tables:
+`sale.group_mean`, `sale.parameter`, and `sale.metadata`.
+
+**Primary Key**: `meta_sale_document_number`, `run_id`, `version`
```

Comment:
I really like having the docs list the primary key somewhere, but I'm somewhat ambivalent about the format I've used here.

One alternative is to add the primary key as a `meta` item for each model:

```yaml
model:
  - name: test
    meta:
      pk:
        - meta_sale_document_number
        - run_id
        - version
```

It then gets formatted into a kinda ugly string in the **Details** section of the docs, like this:

```
'["meta_sale_document_number", "run_id", "version"]'
```

@jeancochrane @wagnerlmichael What do you all think?

Reply:
Totally, that works for me!